dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/validation/DefaultTrackerValidationServiceTest.java

```diff
@@ -60,7 +60,6 @@
 import org.hisp.dhis.tracker.report.ValidationErrorReporter;
 import org.hisp.dhis.tracker.validation.hooks.AbstractTrackerDtoValidationHook;
 import org.hisp.dhis.user.User;
-import org.jetbrains.annotations.NotNull;
```

Comment:
should we ban this one? It was confusing to me to get these different NotNull;NonNull annotations suggested by the IDE.

Reply:
Done :)